packages/core/admin/admin/src/content-manager/pages/EditView/components/FormInputs/Relations.tsx

```diff
@@ -748,7 +748,7 @@ const RelationsList = ({
       }
     }
 
-    addFieldRow(`${name}.disconnect`, { documentId: relation.documentId });
+    addFieldRow(`${name}.disconnect`, relation);
```

Comment:
could we do this?
```suggestion
    addFieldRow(`${name}.disconnect`, { id: relation.id });
```

The "status" meaning difference between Admin and the backend could make some scenarios incompatible, and users could end up in scenarios where they can not disconnect a relation. Sending just the id would be the safest option.

Reply:
do we want to do this with connect as well?

Reply:
Connect should be ok but  lets do that aswell so we keep it  consistent until we are sure about the solution

Reply:
If we only send the id then, isn't your other PR not needed then?

Reply:
We will need it at some point, and it will also be useful for the content api users